src/clr-angular/data/datagrid/datagrid-row.ts

```diff
@@ -258,4 +260,20 @@ export class ClrDatagridRow<T = any> implements AfterContentInit, AfterViewInit
   public get _view() {
     return this.wrappedInjector.get(WrappedRow, this.vcr).rowView;
   }
+
+  @Input('ariaTextBeginningOfExpandableContent') ariaTextBeginningOfExpandableContent: string;
```

Comment:
Why do we need another input property here? Users can update these default strings through CommonStrings, right?

Reply:
You could have multiple datagrids with deferent purposes.

Datagrid A
  - Begining of User details content
  - Content
  - End of User details content

Datagrid B
 - Begining of invoice
 - Content
 - End of invoice

Reply:
I get that there could be multiple datagrids. But I don't see why they don't would want to customize these default strings each one of them? If they need to do that, they could interject the common string service at their component level.

Reply:
This is part of the a11y ask, that the messages are too generic and may need to be tailored to each row or datagrid. This is where we're really hitting the limits with what we can do with a11y.

Reply:
Having a new instance of the `ClrCommonStrings` won't let you change the value row-by-row also if you want to do it datagrid-by-datagrid you will need to create a wrapper for them. 

This way you could have better control.

Reply:
Why do you expect users to want to have custom messages in each row or datagrid? Having different messages on each row would defeat its original purpose. Rows should have consistent header and footer messages to be helpful. You can have a wrapper component or simply place a directive that injects their own custom common strings. 

But besides that, we need to remember we are adding this fix to "deprecated" `clr-dg-row-detail`. So we should really try to keep its changes within itself instead adding a new API property to `clr-dg-row`. We are just expanding the area of the potential breaking change if we ever decides to delete the row detail.

Reply:
As mention above this is an a11y request